apps/full-site-editing/.eslintrc.js

```diff
@@ -5,4 +5,12 @@ module.exports = {
 		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
 		'react/react-in-jsx-scope': 0,
 	},
+	overrides: [
+		{
+			files: [ './**/?(*.)spec.[jt]s?(x)', './full-site-editing-plugin/e2e-test-helpers/**' ],
+			globals: {
+				page: 'readonly',
```

Comment:
I'm curious about the purpose of this setting?

Reply:
If we don't set this rule, then eslint throws an error when you try to access the `page` object from puppeteer. This tells eslint that the page global object is always available in the e2e tests (which it is)